pyproject.toml

```diff
@@ -119,21 +115,24 @@ extend-exclude = '''
 '''
 
 
-
 [tool.ruff]
-exclude = [
-  "*/__init__.py"
-]
 
 [tool.ruff.lint]
-select = ["F401", "I"]  # TODO: eventually, expand to other 'F' linting
+select = ["F401", "I", "D101"] # TODO: eventually, expand to other 'F' linting
+ignore = ["D100"]
```

Comment:
This being the key change

Reply:
Yes, sorry, did format the pyproejct.toml by mistake, it should be reverted.

Reply:
Yes, D100 adds checks that classes have docstrings, and rule D100 is that modules should have docstrings which we skip-ignore.

Reply:
And the plan is to add more pyflakes (F) and pydocstyle (D) rules in follow-up PR(s)?

Reply:
Yes.